Makefile

```diff
@@ -25,6 +25,9 @@ export AEVM_EXTERNAL_TEST_DIR=aevm_external
 HTTP_APP = apps/aehttp
 SWTEMP := $(shell mktemp -d 2>/dev/null || mktemp -d -t 'mytmpdir')
 
+## Dependency `rocksdb` takes long to build, the others are just to shave off some seconds...
+CLEAN_DEPS_EXCEPTIONS=rocksdb sha3 enacl erlexec
```

Comment:
This seems dangerous to me. If we update one of them, we won't notice and it only takes a few seconds.

This optimation takes more time than it saves if we in a few weeks start using a new version of erlexec and people do a make clean to get the latest version in their env.

Reply:
Are you talking about erlexec in particular or about all of them (sha3, enacl and erlexec) ?

Reply:
I somehow agree with @ThomasArts `make clean` is used to clean **all** the deps.
However, we should not speedup the *build after clean*, but we rather should fix all the broken makefiles.

`make any-target` should perfectly work regardless if you updated `erlexec` (or any other vendor lib) or not.

However, this is just my rant about our builds, not strictly related to this PR.